### PR TITLE
get csv from s3

### PR DESCRIPTION
**Pull request by harrybpc**

### CODE REVIEW

File: src/location-input/zipData/fetchZipCodes.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+const colNameLookup: { [key in StoredZipCSVColumnName]: StoredZipDataItemKey } =
+  {
+    state_id: "stateShort",
+    zip: "zip",
+    Availability: "availability",
+    serving_now: "servingNow",
+  };
+
+const unmarshall = (csv: string): StoredZipData => {
```
Comment (scariabpc) [inline src/location-input/zipData/fetchZipCodes.ts]: nit: typo?

File: src/location-input/zipData/zipStore.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export const getZipStore = (sheetConfig: SheetDataConfig) => {
   const load = async () => {
     try {
       const res = await fetchZipCodes(sheetConfig);
+      console.log("response: ", res);
```
Comment (march08) [inline src/location-input/zipData/zipStore.ts]: I'm not sure ATM if the build removes console logs but i'd remove this

**Review by march08 — APPROVED** [review]
lgtm!

File: src/location-input/zipData/zipStore.ts (hunk 2 above)
Comment (harrybpc) [reply src/location-input/zipData/zipStore.ts]: thank you will do

**Pull request merged**